src/modules/meals.js

```diff
@@ -0,0 +1,14 @@
+import $, { data, status } from 'jquery';
```

Comment:
- Kudos 👍  to including JQuery library in your JS. Even when it is not in the requirements, you figured it out.